quizbizz/src/pages/quizinstruction/QuizInstruction.jsx

```diff
@@ -0,0 +1,146 @@
+      showToast("Something Went Wrong", "ERROR");
+    }
+  };
+  return (
+    <div>
+      <Navbar />
+      <main className="page-container">
+        <Sidebar />
+        <div className="col-2 hero-section-container">
+          <div className="quiz-heading-container">
+            <div className="wrapper">
+              <h1 className="h4">{quizDetails.quizName} Quiz</h1>
+              <p className="font-medium primary-text-color">
+                Read the following instructions carefully
+              </p>
+            </div>
+            <div className="image-container">
+              <img
+                src="/images/css.png"
+                className="img-responsive quiz-info-image"
+                alt="quiz-name"
+              />
+            </div>
+          </div>
+
+          {/* <!--  ******************************** Quiz Image and details container Start ********************************  --> */}
+          <div className="quiz-info-container">
+            <div className="image-container">
+              <img
+                src={quizDetails.quizImage}
+                className="img-responsive quiz-info-image"
+                alt="quiz-name"
+              />
+            </div>
+            <div className="quiz-info">
+              <div className="spacer-1"></div>
+              <div className="quiz-info-item">
+                <span className="font-medium-large weight-bold flex-1 margin-horizontal-lg">
+                  Date:
+                </span>
+                <span className="font-medium-large weight-normal flex-1 margin-horizontal-lg">
+                  {todaysDate}
+                </span>
+              </div>
+              <div className="spacer-1"></div>
+              <div className="quiz-info-item">
+                <span className="font-medium-large weight-bold flex-1 margin-horizontal-lg">
+                  Time Limit:
+                </span>
+                <span className="font-medium-large weight-normal flex-1 margin-horizontal-lg">
+                  30 min
+                </span>
+              </div>
+              <div className="spacer-1"></div>
+              <div className="quiz-info-item">
+                <span className="font-medium-large weight-bold flex-1 margin-horizontal-lg">
+                  Attempts:
+                </span>
+                <span className="font-medium-large weight-normal flex-1 margin-horizontal-lg">
+                  3
+                </span>
+              </div>
+              <div className="spacer-1"></div>
+              <div className="quiz-info-item">
+                <span className="font-medium-large weight-bold flex-1 margin-horizontal-lg">
+                  Points:
+                </span>
+                <span className="font-medium-large weight-normal flex-1 margin-horizontal-lg">
+                  200
+                </span>
+              </div>
+              <div className="spacer-1"></div>
+            </div>
+          </div>
```

Comment:
Suggestion: you can make a separate component for display quiz instruction here just because code lines going 100+

Reply:
All the info is static that is why i have not broken the file into components, and these wont be used anywhere this is the reason i have not broken it into components